sagemaker_edge_manager/sagemaker_edge_example/sagemaker_edge_example.ipynb

```diff
@@ -231,7 +241,7 @@
     "- CloudWatchAgentAdminPolicy \n",
     "\n",
     "\n",
-    "There is a `AmazonSageMaker-ExecutionPolicy` attached to the sagemaker role, replace the policy content with below statement:\n",
+    "Locate the same sagemaker role in [Demo Setup](#Demo-Setup) in [IAM console](https://console.aws.amazon.com/iam), find the `AmazonSageMaker-ExecutionPolicy` attached to this sagemaker role, replace the policy content with below statement:\n",
```

Comment:
How do you find this? 
What I tried: Went to IAM roles -> clicked on SM role I am using for this notebook -> scrolled through policies